net/csshnpd/Makefile

```
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2025 The Atsign Foundation
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=csshnpd
+PKG_VERSION:=1.0.8
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-c$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/atsign-foundation/noports/releases/download/c$(PKG_VERSION)
+PKG_HASH:=f579601e47c766ea1531b5d98702c6d7111c700575ae11360b2672c9f9b39d91
+
+PKG_MAINTAINER:=Chris Swan <chris@atsign.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENCE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-c$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_OPTIONS += \
+	-S . \
+	-DATSDK_BUILD_TESTS="OFF" \
+	-DATSDK_MEMCHECK="OFF" \
+	-DBUILD_SHARED_AND_STATIC_LIBS="OFF" \
+	-DCJSON_BUILD_SHARED_LIBS="OFF" \
+	-DCJSON_OVERRIDE_BUILD_SHARED_LIBS="ON" \
+	-DCMAKE_C_FLAGS="-std=c99 -Wno-error -Wextra -Werror-implicit-function-declaration -fPIC -fhonour-copts" \
+	-DCMAKE_EXPORT_COMPILE_COMMANDS="OFF" \
+	-DCMAKE_INSTALL_PREFIX="${PKG_BUILD_DIR}/build/release-static/tmp-install-dir" \
+	-DENABLE_CJSON_TEST="OFF" \
+	-DENABLE_CJSON_VERSION_SO="OFF" \
+	-DENABLE_PROGRAMS="OFF" \
+	-DENABLE_TARGET_EXPORT="OFF" \
+	-DENABLE_TESTING="OFF" \
+	-DNOPORTS_MBEDTLS_PATH="deps/mbedtls-src" \
+	-DNOPORTS_CJSON_PATH="deps/cjson-src" \
+	-DNOPORTS_ATSDK_PATH="deps/atsdk-src"
+
+define Package/csshnpd
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=NoPorts Daemon
+	URL:=https://noports.com
+	SUBMENU:=SSH
+endef
+
+define Package/csshnpd/description
+	A small and portable daemon for NoPorts.
+endef
+
+define Package/csshnpd/conffiles
+/etc/config/sshnpd
+endef
+
+define Package/csshnpd/install	
+	$(INSTALL_DIR) \
+		$(1)/usr/bin \
+		$(1)/etc/config \
+		$(1)/etc/init.d
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sshnpd/sshnpd $(1)/usr/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sshnpd/at_activate $(1)/usr/bin/
+	$(INSTALL_BIN) files/at_enroll.sh $(1)/usr/bin/
+	$(INSTALL_BIN) files/csshnpd.init $(1)/etc/init.d/sshnpd
+	$(INSTALL_CONF) files/csshnpd.config \
+		$(1)/etc/config/sshnpd
+endef
```

```suggestion
define Package/csshnpd/install	
	$(INSTALL_DIR) $(1)/usr/bin
	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sshnpd/sshnpd $(1)/usr/bin/
	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sshnpd/at_activate $(1)/usr/bin/
	$(INSTALL_BIN) ./files/at_enroll.sh $(1)/usr/bin/

	$(INSTALL_DIR) $(1)/etc/config 
	$(INSTALL_BIN) ./files/csshnpd.init $(1)/etc/init.d/sshnpd
	
	$(INSTALL_DIR) $(1)/etc/init.d	
	$(INSTALL_CONF) ./files/csshnpd.config $(1)/etc/config/sshnpd
endef
```

This makes it more readable. :) Nothing else.